README.md

```diff
@@ -15,7 +15,7 @@
 <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/intent/follow?screen_name=botonic_"><img src="https://camo.githubusercontent.com/9e04e9647d574f9e2ad27f93d6eade8f5e9d6b9fed6c50ad6399742b7713ba1f/68747470733a2f2f696d672e736869656c64732e696f2f747769747465722f666f6c6c6f772f626f746f6e69635f3f7374796c653d736f6369616c" alt="Twitter Follow" data-canonical-src="https://img.shields.io/twitter/follow/botonic_?style=social" style="max-width:100%;"></a>
 
 
-# 🐣 What is Botonic?
+# 🐣 What is Botonic Open Source Framework?
```

Comment:
Would it be better to add the open source inside the description rather than in the title?
Something like this:

> [Botonic](https://botonic.io) is an open source full-stack framework to create chatbots...

I understand that this change has been made for a better SEO position, but for me it looks a little odd to have part of the answer in the title itself. What do you think?

Reply:
I agree. Other suggestion: instead of a question, what about "Introducing Botonic Open Source Framework"?

Reply:
If it's good to have this "open source" concept in the title for SEO improvements, then yes, I agree with you and I think that it's better to put it as you suggest, without being a question.

Reply:
Maybe I would also add the "open source" concept into the description, as I suggested in the comment before, what do you think? 
Right now I don't see anywhere in the README file that the framework is open source and I think we should have it in the description also.

Reply:
@asastre Totally, I'm adding it right now :)